### DISCUSSION TITLE
Markdown Headers optionally generated as text in varied size.

### DISCUSSION THREAD

**Discussion by Jonathan-Zollinger** [charmbracelet/glow#469] [Ideas]
## TL;DR

What do we think about adding an option to use ascii text and display headers as different sizes in glow? 

## Longer Version:

I recognize that this may not be appealing to everyone, but enabling a size difference in glow's rendering of the markdown headers I think would be an awesome addition. I dont THINK there's a way to natively change text size in a terminal like you can change color, BUT I thought using ascii text could be a nice workaround. This is a total shot in the dark, it may be that my brain may just works differently (ADHD does that sometimes 😛 ). I like using markdown because of the visual distinction in sections. I love working from the terminal. It'd be awesome if I could have my cake and eat it too!

### Example of IDE-rendered headers vs proposed terminal renderings
<ul><details><summary>Example of H1, H2 and H6 in that order</summary>

# The largest heading
###### The smallest heading
</details></ul>

I used fonts  `big` `small` and `mini` from a[ python based website](https://ascii-generator.site/t/) for this demonstration. the text is unaltered, though I'd like to increase spacing between words if I'm being honest.

<ul><details><summary>Example of 3 varied headers in descending order </summary>

<span style="color:blue">

<br>it seems I can't figure how to get the preview below to show color output (in an effort to mimic glow's default  color style). Pretend this is all colored properly.

```
_______  _             _                                 _     _                       _  _               
|__   __|| |           | |                               | |   | |                     | |(_)              
   | |   | |__    ___  | |  __ _  _ __   __ _   ___  ___ | |_  | |__    ___   __ _   __| | _  _ __    __ _ 
   | |   | '_ \  / _ \ | | / _` || '__| / _` | / _ \/ __|| __| | '_ \  / _ \ / _` | / _` || || '_ \  / _` |
   | |   | | | ||  __/ | || (_| || |   | (_| ||  __/\__ \| |_  | | | ||  __/| (_| || (_| || || | | || (_| |
   |_|   |_| |_| \___| |_| \__,_||_|    \__, | \___||___/ \__| |_| |_| \___| \__,_| \__,_||_||_| |_| \__, |
                                         __/ |                                                        __/ |
                                        |___/                                                        |___/ 
 _____  _                                      _   _                            _     _                   _  _             
|_   _|| |_   ___   ___ ___  __  ___  _ _   __| | | | __ _  _ _  __ _  ___  ___| |_  | |_   ___  __ _  __| |(_) _ _   __ _ 
  | |  | ' \ / -_) (_-</ -_)/ _|/ _ \| ' \ / _` | | |/ _` || '_|/ _` |/ -_)(_-<|  _| | ' \ / -_)/ _` |/ _` || || ' \ / _` |
  |_|  |_||_|\___| /__/\___|\__|\___/|_||_|\__,_| |_|\__,_||_|  \__, |\___|/__/ \__| |_||_|\___|\__,_|\__,_||_||_||_|\__, |
                                                                |___/                                                |___/ 

___                                                                   
 |  |_   _     _ ._ _   _. | |  _   _ _|_   |_   _   _.  _| o ._   _  
 |  | | (/_   _> | | | (_| | | (/_ _>  |_   | | (/_ (_| (_| | | | (_| 
                                                                   _| 

```
</span>
</details></ul>
